source/scss/_library/_module.header.scss

```diff
@@ -4,6 +4,187 @@
 
 .c-main-header {
   text-align: center;
-  background-color: $c-secondary;
-  padding-top: $pad;
+  padding-top: 5px;
+
+  &__ad-unit {
+    position: relative;
+    z-index: 100;
+    background-color: $c-white;
+  }
+
+  &__inner {
+    background-color: $c-heading-bgcolor;
+    padding: $space-half $space-half $space-half 50px;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    position: relative;
+    height: 70px;
+
+    @include media('>xsmall') {
+      padding: $space-half $space $space-half 70px;
+    }
+  }
+
+  &__branding {
+    width: 120px;
+    transform: translateY(5px);
+
+    @include media('>xsmall') {
+      width: 150px;
+    }
+  }
+
+  &__right {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    position: relative;
+    height: 100%;
+  }
+
+  &__donate {
+    margin-right: $space;
+    @include media('<=small') {
+      margin-right: 0;
+      font-size: 12px;
+      padding-left: $pad;
+      padding-right: $pad;
+    }
+  }
+}
+
+.c-search-toggle {
+  width: 40px;
+  height: 40px;
+  transform: translate(-5px, 2px);
```

Comment:
see my note about making this a quadrinary button. I want to think about this button as a button that's like any other button that just happens to be placed next to a search input, which means that something like this rule would not be portable to other spots in the design.

Reply:
It's not really a "button" as in style, but a "button" as in the element. This button triggers the flyout of the search form. 

![img](https://d.pr/i/InqVVi+)